evalml/tests/utils_tests/test_woodwork_utils.py

```diff
@@ -203,3 +208,50 @@ def test_infer_feature_types_raises_invalid_schema_error():
         df.ww.init()
         df.drop(columns=["b"], inplace=True)
         infer_feature_types(df)
+
+
+def test_ordinal_retains_order_min():
+    features = pd.DataFrame(
+        {
+            "non-ordinal": [0, 1, 2, 3, 4, 5],
+            "ordinal": [0, 1, 2, 2, 1, 0],
+            "categorical": ["red", "white", "blue", "red", "white", "blue"],
+            "datetime": [
+                "2020-09-10",
+                "2020-09-11",
+                "2020-09-12",
+                "2020-09-13",
+                "2020-09-14",
+                "2020-09-15",
+            ],
+        }
+    )
+    logical_types = {
+        "non-ordinal": "Age",
+        "ordinal": Ordinal(order=[0, 1, 2]),
+        "categorical": Categorical(encoding="Encoding"),
+        "datetime": Datetime(datetime_format="%Y-%m-%d"),
+    }
+    features.ww.init(logical_types=logical_types)
+
+    # Ordinal type should now pass through the function without issue and retain the 'order' property
+    ordinal_subset = _retain_custom_types_and_initalize_woodwork(
+        old_logical_types=logical_types, new_dataframe=features[["ordinal"]]
+    )
+    ltypes = ordinal_subset.ww.logical_types
+    assert ltypes["ordinal"].order is not None
```

Comment:
Can we not assert that this order is `[0, 1, 2]`? Or do the new logical types not use the old logical types?